engines/supernova/supernova.h

```diff
@@ -137,6 +137,8 @@ class SupernovaEngine : public Engine {
 	void renderBox(const GuiElement &guiElement);
 	void setColor63(byte value);
 	void stopSound();
+	// Common::Language getLanguage() const;
+	Common::TextToSpeechManager *_ttsMan;
```

Comment:
I don't see the benefit of having it as a class member variable since it is only used once in one place. I would suggest to use a local variable in `SupernovaEngine::run()` instead.